### ISSUE TITLE
issues with Lumen

### ISSUE THREAD

**Issue by sylvaincherrier** [elibyy/tcpdf-laravel#58]
Thx for your work !
it really helps. I had some issues to make it work with Lumen.
i had to create a helpers.php file in app/
(with this content) :
 

```
<?php
  if ( ! function_exists('config_path'))
    /**
       * Get the configuration path.
       *
       * @param  string $path
       * @return string
       */
     function config_path($path = '')
      {
          return app()->basePath() . '/config' . ($path ? '/' . $path : $path);
      }
  }
```

 and to modify composer.json to add 
>,
>        "files": [
>            "app/helpers.php"
>        ]

Then, it works.
Am i the only one ? did i miss something ?
Thx

**Comment by elibyy** [comment]
Hey there,
It seems that you are right, this will be fixed very soon.
Thanks

**Comment by elibyy** [comment]
Fixed in [5.5.5](https://github.com/elibyy/tcpdf-laravel/releases/5.5.5) or [5.6.3](https://github.com/elibyy/tcpdf-laravel/releases/5.6.3)

**Comment by elibyy** [comment]
This issue is being closed due to no further response
